docs/getting-started/supported-components.md

```diff
@@ -0,0 +1,125 @@
+# Supported Components
+
+The following list of Material Design components and features highlights 
+those currently supported by Material-UI in bold.
```

Comment:
Well, sure, but my point was that **bold** is no substitute for the GutHub Markdown check-list, especially for the colored links. Is thre a better alternative?